mappings/net/minecraft/util/math/Quaternion.mapping

```diff
@@ -0,0 +1,31 @@
+CLASS net/minecraft/class_1158 net/minecraft/util/math/Quaternion
+	FIELD field_5656 components [F
+	METHOD <init> (FFFF)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (FFFZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 w
+	METHOD <init> (Lnet/minecraft/class_1158;)V
+		ARG 1 other
+	METHOD <init> (Lnet/minecraft/class_1160;FZ)V
+		ARG 1 direction
+		ARG 2 magnitude
+		ARG 3 normalize
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
+	METHOD method_16002 fSin (F)F
+		ARG 0 value
+	METHOD method_16003 fCos (F)F
+		ARG 0 value
```

Comment:
What does `f` mean in `fSin` and `fCos`?

Reply:
If it means that the methods work on floats, then just `sin` and `cos` is enough, the type is already in the method signature, there's no need to specify it in the name too. `f` is confusing (it could also mean "fast", for example).

Reply:
It's to be in line with the GlStateManager methods, which have `scalef` for example.

Reply:
What? This is not a `GlStateManager` method. The methods in GlStateManager are named to match the OpenGL method names, which are not overloaded because C doesn't support overloading. And anyway, even if you were naming this as a GlStateManager method, it should be `sinf`, not `fSin`.